### DISCUSSION TITLE
Theme customisation/designer

### DISCUSSION THREAD

**Discussion by v-c0de** [massCodeIO/massCode#63] [Ideas]
Possibility of user's customising and creating their own themes for the editor.

These can then be shared with other users (and also makes it more customisable). 

Example:

https://www.renfei.org/snippets-lab/
![image](https://user-images.githubusercontent.com/65813722/166114812-37dca8ca-7850-431d-a5ac-e9db86dabf07.png)

**Comment by antonreshetov** [comment]
@v-c0de I thought about it, but I did not implement it because honestly I do not see the point. Perhaps I will add more themes in the future
